aws/modules/elasticache_redis/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_elasticache_subnet_group" "ts_aws_elasticache_subnet_group" {
+  name       = "ts-aws-elasticache-redis-subnet-group"
+  subnet_ids = var.ts_aws_elasticache_subnet_group_subnet_ids
+}
+
+resource "aws_elasticache_replication_group" "ts_aws_elasticache_replication_group" {
+  replication_group_id = var.ts_aws_elasticache_replication_group_id
+  description          = var.ts_aws_elasticache_replication_group_description
+
+  node_type            = var.ts_aws_elasticache_replication_group_node_type
+  port                 = var.ts_aws_elasticache_replication_group_port
+  parameter_group_name = var.ts_aws_elasticache_replication_group_parameter_group_name
+
+  snapshot_retention_limit = var.ts_aws_elasticache_replication_group_snapshot_retention_limit
+  snapshot_window          = var.ts_aws_elasticache_replication_group_snapshot_window
+
+  subnet_group_name          = aws_elasticache_subnet_group.ts_aws_elasticache_subnet_group.name
+  security_group_ids         = var.ts_aws_elasticache_replication_group_security_group_ids
```

Comment:
Rather than requiring the user to pass in the security group ids, it would be more usable to create a default security group that allows ingress from the vpc.  Then additional optional security group ids could be passed in to add additional ingress rules allowing traffic from those security groups to the cluster's port.

Reply:
So basically do you mean we need to some how merge the security groups with the default one created along with additional ones they create?

The reason I added it to the stitcher repo and not within it because it needs the VPC anyway to create the security group also this keeps the logic simpler if folks want to add additional groups.

Reply:
Done.